seaworthy/tests-core/test_dockerhelper.py

```diff
@@ -131,3 +131,101 @@ def test_teardown_containers(self):
             "Container 'test_stopped' still existed during teardown",
         ])
         self.assertEqual([], self.list_containers(all=True))
+
+    def test_create_container(self):
+        """
+        We can create a container with various parameters without starting it.
+        """
+        dh = self.make_helper()
+
+        con_simple = dh.create_container('simple', IMG)
+        self.addCleanup(dh.remove_container, con_simple)
+        self.assertEqual(con_simple.status, 'created')
+        self.assertEqual(con_simple.attrs['Path'], 'nginx')
```

Comment:
This `attrs` thing... have I been doing `_client.api.inspect(container.id)` for no reason?

Reply:
Perhaps? I haven't looked at the inspect stuff yet. Maybe you get a nicer format or something.

Reply:
It does look like it has the contents of the inspect call, which means we can clean some things up a bit.

Reply:
Yup, `.attrs` is populated directly from `api.inspect()` and all the other container properties just read from that.